crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -368,34 +368,6 @@ impl Entities {
         }
     }
 
-    /// Panics if the given id would represent an index outside of `meta`.
-    ///
-    /// # Safety
-    ///
-    /// Must only be called for currently allocated `id`s.
-    pub unsafe fn resolve_unknown_gen(&self, id: u32) -> Entity {
```

Comment:
Hmm, this might conflict with #2581

I would also note that this method doesn't actually perform any unsafe operations (or put us in an inconsistent state), which is kind of fun

Reply:
Removed this commit.